src/domains/Feedback/ReportHistory/ReportHistory.tsx

```diff
@@ -8,15 +8,16 @@ import Select from '@material-ui/core/Select';
 import Toolbar from '@material-ui/core/Toolbar';
 import Grid from '@material-ui/core/Grid';
 import Button from '@material-ui/core/Button';
-import SortIcon from '@material-ui/icons/Sort';
-import SearchIcon from '@material-ui/icons/Search';
-import ArrowDownIcon from '@material-ui/icons/ArrowDropDown';
+import { Sort as SortIcon, Search as SearchIcon, ArrowDropDown as ArrowDownIcon } from '@material-ui/icons';
```

Comment:
Just be aware that doing these types of imports requires an optimization: https://material-ui.com/guides/minimizing-bundle-size/#minimizing-bundle-size.  I've already made this optimization, but if I hadn't then this would lead to much longer build times.  This comment is more so for educational purposes.

Reply:
Oh didn't know. I just updated like that to group icon imports together. Thanks for the information!